### ISSUE TITLE
children image files not found

### ISSUE THREAD

**Issue by ShKlinkenberg** [rstudio/rmarkdown#1139]
This version of rmarkdown is showing problems with images contained in child files using ioslides.

[children.zip](https://github.com/rstudio/rmarkdown/files/1287991/children.zip)

I'm having no problem parsing the Children.Rmd file in presentation (see zip) with the cran version 1.6 but version 1.6.09 is not displaying the images that are plotted.

**Comment by caesarivs** [comment]
If you add the `self_contained: false` parameter on the YAML, the images are displayed, but a separate `<presentation_name>_files` directory is created.

With default YAML the images are not displayed but the HTML expects to find them at said directory.

**Comment by atusy** [comment]
Seems like the bug is fixed already by rmarkdown 2.3.2 

I even tried rmarkdown 1.6, but did not reproduce the bug.
Maybe something has changed on Pandoc side?

<details><summary> xfun::session_info('rmarkdown')</summary>
```
R version 4.0.2 (2020-06-22)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04 LTS, RStudio 1.3.1056

Locale:
  LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8       
  LC_COLLATE=en_US.UTF-8     LC_MONETARY=en_US.UTF-8    LC_MESSAGES=C             
  LC_PAPER=en_US.UTF-8       LC_NAME=C                  LC_ADDRESS=C              
  LC_TELEPHONE=C             LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

Package version:
  base64enc_0.1.3 digest_0.6.25   evaluate_0.14   glue_1.4.1      graphics_4.0.2  grDevices_4.0.2
  highr_0.8       htmltools_0.5.0 jsonlite_1.7.0  knitr_1.29      magrittr_1.5    markdown_1.1   
  methods_4.0.2   mime_0.9        rlang_0.4.7     rmarkdown_2.3.2 stats_4.0.2     stringi_1.4.6  
  stringr_1.4.0   tinytex_0.24    tools_4.0.2     utils_4.0.2     xfun_0.15       yaml_2.2.1     

Pandoc version: 2.7.3
```
</details>

**Comment by yihui** [comment]
I'm unable to reproduce the issue, either. It may indeed have been fixed in a certain version of Pandoc, although I don't know which... Thanks!
